src/components/Pagination/index.stories.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-return-assign */
+
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+
+import Heading from '../Heading';
+import Pagination from './index';
+
+const stories = storiesOf('Pagination', module);
+
+stories.add('Playground', () => {
+  const [page, setPage] = useState(0);
+  return (
+    <>
+      <Heading>Pagination</Heading>
+      <Heading level={2}>Page #{page}</Heading>
+
+      <Pagination
+        prevDisabled={page === 0}
+        nextDisabled={page === 3}
```

Comment:
How would you implement that in a real-life situation?

Reply:
You can check the `hashtag management` page or the `vouchers` in the admin for its usage. 

But basically it depends on your factors. Sometimes the logic comes from the values in a data object, sometimes from something else.  The component doesn't know what page it's on, hence the need for it

Reply:
I see 🤔 
Wouldn't it be simpler to pass a `currentPage` and a `totalPage` sort of props?

Reply:
indeed, but we don't always know how many pages there are tho.

Reply:
for example in `user search` the logic for pagination is special. We don't know how many pages there are but each page has 100 items. So the `nextDisabled === { if current page has less than 100 items}`